InfiniteLoopers-App/src/main/java/com/isa/account/User.java

```diff
@@ -104,6 +104,14 @@ public void setUserRole(UserRole userRole) {
 
     @Override
     public String toString() {
+        StringBuilder sb = new StringBuilder();
+        sb.append("Класс: ").append(this.getClass().getSimpleName()).append("\n");
+        sb.append("Оценки:\n");
```

Comment:
Myślę, że reszta może nie zrozumieć tych komunikatów:)

Reply:
No tak. W ogóle to nie jest potrzebne w projekcie